frontend/src/metabase/querying/notebook/components/NotebookDataPicker/EmbeddingDataPicker/EmbeddingDataPicker.tsx

```diff
@@ -108,8 +108,6 @@ export function EmbeddingDataPicker({
       databaseQuery={{ saved: true }}
       canSelectModel={entityTypes.includes("model")}
       canSelectTable={entityTypes.includes("table")}
-      canSelectMetric={false}
-      canSelectSavedQuestion={false}
```

Comment:
We passed these props when working on the new data picker. They were for the other picker that might still want to show metrics or saved questions, but not for the embedding data picker.